baselines/dom.generated.d.ts

```diff
@@ -7112,7 +7112,7 @@ interface IDBRequestEventMap {
 }
 
 interface IDBRequest extends EventTarget {
-    readonly error: DOMError;
+    readonly error: DOMException | DOMError;
```

Comment:
Structurally DOMException is a subtype of DOMError so it shouldn't cause any compatibility issue.